ruby/vampires.rb

```diff
@@ -0,0 +1,66 @@
+puts "How many employees will be processed?"
+employee = gets.to_i
+
+vampire = 0
+
+#Loop until employee processing is met.
+until vampire == employee 
+
+	puts "What is your name?"
+	name1 = gets.chomp
+    	if name1 == "Drake Cula"
+    		name2 = false
+    	elsif name1 == "Tu Fang"
+    		name2 = false
+    		else name2 = true
+    	end
+  
+    puts "How old are you? What year were you born?"
+	age = gets.to_i
+	birth_year = gets.to_i
+  		if birth_year + age == 2016
+  			age2 = true
+  			else age2 = false
+  		end
+		
+	puts "Our company cafeteria serves garlic bread. Should we order some for you? (y/n)"
+	garlic = gets.chomp
+    	if garlic == "y"
+    		garlic2 = true
+    		else garlic == "n"
```

Comment:
`else` does not take a condition.  `garlic == "n"` is not being ran the way you may think.  `else` will run if all other conditions do not pass.  If you want another condition then consider `elsif`.  In this case you are fine just using `else` without the condition.